app/assets/javascripts/student_profile/note_card.jsx

```diff
@@ -169,6 +191,60 @@ import moment from 'moment';
           (remove)
         </a>
       );
+    },
+
+    renderHomeroomOrGrade: function(student) {
+      if (student.grade < 9) {
+        if (student.homeroom_id) {
+          return (
+            <p><a
+              className="homeroom-link"
+              href={Routes.homeroom(student.homeroom_id)}>
+              {'Homeroom ' + student.homeroom_name}
+            </a></p>
+          );
+        }
+        else {
+          return (
+            <p>No Homeroom</p>
+          );
+        }
+      }
+      else {
+        return (
+          <p>{student.grade}th Grade</p>
+        );
+      }
+    },
+
+    renderSchool: function(student) {
+      if (student.school_id) {
+        return (
+          <p><a
+            className="school-link"
+            href={Routes.school(student.school_id)}>
+            {student.school_name}
+          </a></p>
+        );
+      }
+      else {
+        return (
+          <p>No School</p>
+        );
+      }
+    },
+
+    renderStudentCard: function() {
+      const student = this.props.student;
+      if (student) {
+        return (
+          <div className="studentCard" style={styles.studentCard}>
+            <p style={styles.studentName}>{student.last_name}, {student.first_name}</p>
```

Comment:
Talking about this with @gabivoicu, we're thinking this should link to the student profile.